README.md

```diff
@@ -100,35 +94,33 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+<!-- > Describe between 1-3 key features of the application. -->
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Create animals table**
+- **Populate the table**
+- **Perform some queries to retrieve specific data or alter the database**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
 > Add a link to your deployed project.
 
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Live Demo Link](<replace-with-your-deployment-URL>) -->
```

Comment:
- [ ] [OPTIONAL AND IMPORTANT] as this project does not require deployment, kindly remove the live demo section